From ddfdbc58066eea988978c7943709e6a3b6376deb Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Mon, 11 Dec 2023 14:30:43 +0100
Subject: [PATCH 0715/1141] arm64: dts: st: add sleep pins for pcie in
 stm32mp257f-ev1

Add analog gpio pin configuration ("sleep") to manage power mode on
stm32mp25.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I36628562e5f2b1d5ccf8a6a5b4469c41cc0ce0b7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/345512
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 7051cdd884d3..7c768ca09759 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -537,9 +537,10 @@ &pcie_ep {
 };
 
 &pcie_rc {
-	pinctrl-names = "default", "init";
+	pinctrl-names = "default", "init", "sleep";
 	pinctrl-0 = <&pcie_pins_a>;
 	pinctrl-1 = <&pcie_init_pins_a>;
+	pinctrl-2 = <&pcie_sleep_pins_a>;
 	reset-gpios = <&gpioj 8 GPIO_ACTIVE_LOW>;
 	status = "okay";
 };
-- 
2.39.2

